package com.tuempresa.tresfxml.application;

import javafx.application.Application;
import javafx.fxml.FXMLLoader;
import javafx.scene.Scene;
import javafx.stage.Stage;

import java.io.IOException;

public class MenuApplication extends Application {
    @Override
    public void start(Stage stage) throws IOException {

        FXMLLoader fxmlLoader = new FXMLLoader(MenuApplication.class.getResource("/com/tuempresa/tresfxml/Menu.fxml"));
        Scene scene = new Scene(fxmlLoader.load());
        stage.setTitle("Registro de Cliente");
        stage.setScene(scene);
        stage.show();
    }
}
